### PR TITLE
Tw 1827/update online status based on design

### PR DESCRIPTION
**Pull request by sherlockvn**
### Description:
#1827 

### Design: 
https://www.figma.com/design/uLw3iWnOEGaRlJnfPz4Pv0/Twake-%5BMatrix%5D?node-id=25784-144000&t=wVc6M5FhcRkM6BLY-4

### Demo IOS:

https://github.com/linagora/twake-on-matrix/assets/43041967/2552d1ee-bcfc-400b-83c9-063858c2b41f

### Demo Android:

https://github.com/linagora/twake-on-matrix/assets/43041967/d81a88b5-8ac6-429e-b9ae-70d31802b7be

### CODE REVIEW

**Comment by nqhhdev** [comment]
Widget test build failed, pls check it

File: test/utils/get_localize_status_test.dart [hunk 1]
```diff
@@ -46,12 +46,12 @@ void main() async {
 
       expect(textWidget.data, isNotNull);
 
-      expect(textWidget.data, equals('Active now'));
+      expect(textWidget.data, equals('Online'));
```
Comment (hieutbui) [inline test/utils/get_localize_status_test.dart:49]: How about store expect value in a constant

Comment (sherlockvn) [reply test/utils/get_localize_status_test.dart]: done

**Comment by Te-Z** [comment]
Please add issue to description

**Review by Te-Z — APPROVED** [review]

**Review by nqhhdev — APPROVED** [review]
I tested it on Mobile (both Android and iOS) and web. LGTM

**Review by hieutbui — APPROVED** [review]

**Pull request merged**